Directory.Build.props

```diff
@@ -45,4 +45,10 @@
     </PackageReference>
   </ItemGroup>
 
+  <!-- Needed for reverting back to pre-.NET 8 method of Host using the RID graph to determine assets  -->

```

Comment:
Please make an issue on this repo to amend the code so we don't need the RID workaround.